configs/cascade_mask_rcnn_r50_c4_1x.py

```diff
@@ -0,0 +1,230 @@
+# model settings
+model = dict(
+    type='CascadeRCNN',
+    num_stages=3,
+    pretrained='open-mmlab://resnet50_caffe',
+    backbone=dict(
+        type='ResNet',
+        depth=50,
+        num_stages=3,
+        strides=(1, 2, 2),
+        dilations=(1, 1, 1),
+        out_indices=(2, ),
+        frozen_stages=1,
+        normalize=dict(type='BN', frozen=True),
+        stage_with_dcn=(False, False, False),
```

Comment:
This field can be just omitted.

Reply:
This field can't be omitted. Since it is different from those with fpn.